src/tf/tf.cpp

```diff
@@ -339,6 +316,35 @@ struct tf_parser
             }
         }
 
+        if(contains(attributes, "padding"))
```

Comment:
I have moved this section below strides as a side effect of another branch for testing. It adds another check for "valid" pad_mode, but otherwise is the same.